src/encoded/static/components/facets/defaults.js

```diff
@@ -1010,41 +1010,62 @@ export const DefaultFacet = ({ facet, results, mode, relevantFilters, pathname,
         setInitialState(false);
     }, [handleScrollShading, facet, typeaheadTerm]);
 
+    React.useEffect(() => {
+        setFieldAsNotNewlyLoaded(facet, relevantFilters);
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let me know if this is a concern. I need this to run only once and this was a way to do it and I had to silent ESLint.

Reply:
This code has been removed.